k8s-ingress/services.yaml

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
I'd recommend calling this file `backends.yml` or `apps.yml` or something like that. I was expecting `services.yml` to contain only k8s service objects.